tools/windows-node-installer/pkg/cloudprovider/factory.go

```diff
@@ -84,10 +87,10 @@ func CloudProviderFactory(kubeconfigPath, credentialPath, credentialAccountID, r
 	return nil, err
 }
 
-// makeValidAbsPath remakes a path into an absolute path and ensures that it exists.
+// MakeValidAbsPath remakes a path into an absolute path and ensures that it exists.
 // TODO: Break this function to validate files. dirs etc. As of now, we don't differentiate
 // between files and dirs
-func makeValidAbsPath(path string) (string, error) {
+func MakeValidAbsPath(path string) (string, error) {
```

Comment:
Are you making it public to be used in the test suite?

Reply:
Yes

Reply:
Im of the opinion that we should avoid doing this if possible